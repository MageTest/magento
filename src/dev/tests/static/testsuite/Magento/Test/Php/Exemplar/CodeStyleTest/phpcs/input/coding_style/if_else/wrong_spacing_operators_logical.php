<?php
if ($a||$b) {
    echo 'ab';
}

if ($a|| $b) {
    echo 'ab';
}

if ($a ||$b) {
    echo 'ab';
}

if ($a  || $b) {
    echo 'ab';
}

if ($a ||  $b) {
    echo 'ab';
}
