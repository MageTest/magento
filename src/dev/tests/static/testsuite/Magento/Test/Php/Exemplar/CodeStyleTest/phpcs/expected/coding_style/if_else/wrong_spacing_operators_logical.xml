<?xml version="1.0" encoding="UTF-8"?>
<config>
    <error line="2"/>
    <error line="6"/>
    <error line="10"/>
    <error line="14"/>
    <error line="18"/>
</config>
